cmake/ycm-0.3.1/find-modules/FindZFP.cmake

```diff
@@ -29,25 +29,27 @@ include(SelectLibraryConfigurations)
 
 find_path(ZFP_INCLUDE_DIR
           NAMES zfp.h
-          PATHS $ENV{ZFP_ROOT}/inc
+          PATHS $ENV{ZFP_DIR}/include
```

Comment:
I just noticed this...
Why did you change the name of the environment variable? If there is not a good reason from the upstream changes, the convention is:
ROOT = root of the library installation
DIR = directory containing cmake files